src/main/resources/avro/referencemethods.avdl

```diff
@@ -12,19 +12,16 @@ as JSON.
 */
 record SearchReferenceSetsRequest {
   /**
-  If nonempty, return the reference sets which match any of the given
-  `md5checksum`s. See `ReferenceSet::md5checksum` for details.
+  If not null, return the reference sets which match the given
+  `md5checksum`. See `ReferenceSet::md5checksum` for details.
   */
-  array<string> md5checksums = [];
+  union { null, string } md5checksum = null;
 
   /**
-  If nonempty, return reference sets for which the accession
-  matches this string. Best to give a version number (e.g. `GCF_000001405.26`).
-  If only the main accession number is given then all records with
-  that main accession will be returned, whichever version.
-  Note that different versions will have different sequences.
+  If not null, return reference sets for which the accession
+  matches this string.
```

Comment:
@diekhans, are you describing the existing text, or the new text? I see a mention of version numbering in the existing text, but not in the new text. We should clarify the comment whether or not we end up deciding to change the behavior.

Two fyis:
- I was curious about the history of this part of the query interface -- it looks like it's unchanged since @richarddurbin first submitted it in #66 
- the current Google implementation does a "STARTS_WITH" for `accessions`. (And does an exact match for `md5checksums`.)